Transcelerator/NewQuestionDlg.cs

```diff
@@ -118,22 +108,11 @@ private int EndVerse
 			}
 		}
 
-		public int Category
-		{
-			get { return m_cboCategory.SelectedIndex; }
-		}
+		public int Category => m_cboCategory.SelectedIndex;
 
-		public int SequenceNumber
-		{
-			get
-			{
-				if (m_dataGridViewExistingQuestions.RowCount == 0)
-					return 1;
-				
-				return ((TranslatablePhrase)m_dataGridViewExistingQuestions.Rows[0].DataBoundItem).SequenceNumber +
-					SelectedInsertionLocation;
-			}
-		}
+		public int SequenceNumber =>
+			BasePhrase == null ? 0 :
+				BasePhrase.SequenceNumber + (InsertBeforeBasePhrase ? 0 : 1);

```

Comment:
Since this property now works off of BasePhrase, the behavior has changed since it now looks at the selected rows and also behaves differently when there are more than one. Was this change intentional?

Reply:
The Sequence number used to be a 1-based index for the questions in a particular verse (or verse range). Now it uniquely identifies this phrase within a given category of a particular section. I have added a comment here and updated the comment on TranslatablePhrase.SequenceNumber.